src/D8/MenuLinkContentTrait.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace IntegratedExperts\BehatSteps\D8;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\menu_link_content\Entity\MenuLinkContent;
+
+/**
+ * Trait MenuLinkContentTrait.
+ *
+ * @package IntegratedExperts\BehatSteps\D8
+ */
+trait MenuLinkContentTrait {
+
+  /**
+   * Remove menu.
+   *
+   * Provide menu data in the following format:
+   *
+   * | id        |
```

Comment:
using IDs is a very narrow use case. usually, links are referred by their Titles. Please implement a lookup system by title. If the `id` or `link` is provided - then lookup should be using menu items that match all of the criteria

Reply:
Thanks Alex - this is solid feedback, I've adapted the steps to work on menu titles, and menu link titles so that IDs are not required and not needed by default.